verilog/parser/verilog_token.h

```diff
@@ -0,0 +1,29 @@
+// Copyright 2017-2020 The Verible Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef VERIBLE_VERILOG_TOKEN_VERILOG_TOKEN_H__
+#define VERIBLE_VERILOG_TOKEN_VERILOG_TOKEN_H__
+
+#include <string>
+
+#include "verilog/parser/verilog_token_enum.h"
+
+namespace verilog {
+
+// Stringifier for verilog_tokentype
```

Comment:
Since this is the public header, let's add a little more information for the readers here.  

These strings come from the names of tokens chosen in `verilog/parser/verilog.y`.
This uses the original token names, not their (optional) display names.
(Give a few examples of the strings, like TK_EOL_COMMENT.)
If there are important constraints about the strings, state them here, e.g."these strings should not contain un-escaped quote characters" (or something like that).